src/vivarium_public_health/population/base_population.py

```diff
@@ -18,7 +18,7 @@ class BasePopulation:
             'age_start': 0,
             'age_end': 125,
             'exit_age': None,
-            'population_size': 10000,
+            'population_size': 100,
```

Comment:
This should still fail, I think.  `population_size` is set by the population manager in vivarium.

Reply:
when I ran the pytest before this change, test_base_population failed with the population size 100 set by population manager that was still a component level configs, then base_population tries to override at the same config level.